apps/task.go

```diff
@@ -3,10 +3,13 @@ package apps
 import (
 	"encoding/json"
 	"strings"
+
+	"github.com/allegro/marathon-consul/time"
 )
 
 type Task struct {
 	ID                 TaskID              `json:"id"`
+	Timestamp          time.Timestamp      `json:"timestamp"`
```

Comment:
Pleas add a comment to this. This field is not a part of a Marathon task object. It's only present in the event. We parse task status event to this struct but it may be misleading in the future.